ts/features/bonus/bpd/screens/iban/IbanCTAEditScreen.tsx

```diff
@@ -80,6 +80,7 @@ const InnerIbanCTAEditScreen = (props: Props) => {
         .getOrElse(bpdPeriods, [])
         .find(p => p.status === "Active");
       if (activePeriod) {
+        goBack();
```

Comment:
Yes the behaviour is similar, we need to return to the previous screen before to navigate to the next, otherwise this "dummy" screen remains on the stack.